x-pack/plugins/ml/public/application/components/data_grid/column_chart.scss

```diff
@@ -1,9 +1,9 @@
-.mlDataGridChart__histogram {
```

Comment:
why do these classes need renaming when they are in ML?
The data visualizer should not rely on any styles in ML

Reply:
Thanks for catching that. Reverted the ML changes here https://github.com/elastic/kibana/pull/100922/commits/aaf142c7ebdd590160e216e17a89d1fce0adf656